pkg/batcher/describeinstances.go

```diff
@@ -33,14 +35,14 @@ type DescribeInstancesBatcher struct {
 	batcher *Batcher[ec2.DescribeInstancesInput, ec2.DescribeInstancesOutput]
 }
 
-func NewDescribeInstancesBatcher(ctx context.Context, ec2api ec2iface.EC2API) *DescribeInstancesBatcher {
+func NewDescribeInstancesBatcher(ctx context.Context, ec2Client awsapi.EC2API) *DescribeInstancesBatcher {
```

Comment:
Why rename? Not a huge deal, just doesn't seem like it's tangibly getting us anything here -- also increases the size of the diff